lua/nvim-gps/init.lua

```diff
@@ -13,7 +13,13 @@ local default_config = {
 		["function-name"] = ' ',
 		["method-name"] = ' ',
 		["container-name"] = '⛶ ',
-		["tag-name"] = '炙'
+		["tag-name"] = '炙',
+		["array-name"] = ' ',
```

Comment:
These capture types seem like they would only be useful for JSON. So maybe it would be better to add them language specific configuration (Like the verilog's `module-name`)